src/components/ValuesScreen.tsx

```diff
@@ -1,28 +1,127 @@
 import React from 'react';
-import { View } from 'react-native';
-import { Text } from 'react-native-paper';
+import { View, StyleSheet } from 'react-native';
+import { Text, Title, List, Button} from 'react-native-paper';
 
 import { createStackNavigator } from '@react-navigation/stack';
 import { CustomNavigationBar } from './CustomNavigationBar';
+import { FlatList } from 'react-native-gesture-handler';
 
 const ValuesStack = createStackNavigator();
 
-const ViewContent = () => {
+const RelationView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Relation</Text>
+  </View>
+); 
+
+const CareerView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Studier/karrriär</Text>
+  </View>
+); 
+
+const InterestsView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Fritid/intressen</Text>
+  </View>
+);
+
+const MindView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Sinne/Kropp/Spirituellt</Text>
+  </View>
+);
+
+const ResposibilityView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Dagliga ansvar</Text>
+  </View>
+);
+
+const SupportView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Stödpersoner</Text>
+  </View>
+);
```

Comment:
All of these are exactly the same (except for a string). I have not looked into it much, but I think this is a good use for higher order functions.

Reply:
Yes, these are only temporary views.

Reply:
Tip: If the view are going to be similar it could possibly be made by dynamically generating the view. Something to explore when development continues.

Reply:
Yes we will look into that.